pkg/api/folder.go

```diff
@@ -509,13 +509,6 @@ type UpdateFolderParams struct {
 	Body folder.UpdateFolderCommand `json:"body"`
 }
 
-// swagger:parameters getFolderByID
-type GetFolderByIDParams struct {
-	// in:path
-	// required:true
-	FolderID int64 `json:"folder_id"`
-}
-
```

Comment:
not sure if this is the proper way to delete a struct? it seems regenerating the specs after doing this didn't work

`The swagger spec at "public/api-merged.json" is invalid against swagger specification 2.0.
See errors below:
- path param "{folder_id}" has no parameter definition

make: *** [swagger-validate] Error 1`